app/api/entities/entities.js

```diff
@@ -259,6 +259,30 @@ function sanitize(doc, template) {
   return Object.assign(doc, { metadata });
 }
 
+function applyDiffMetadata(metadata, diffMetadata) {
```

Comment:
I would prefer a more clear name for this function

Reply:
updateMetadataWithDiff?